app/utils/images.py

```diff
@@ -99,6 +100,18 @@ def save(
             duration=duration,
             loop=0,
         )
+    elif extension == "webp":
+        frames, duration = render_animation(
+            template, style, lines, size, font_name, maximum_frames, watermark=watermark
+        )
+        webp_frames = []
+        for frame in frames:
+            # Save the frame as webp
+            f = io.BytesIO()
+            frame.save(f, "webp")
+            webp_frames.append(Image.open(f))
```

Comment:
@Helvio88 Is this loop needed? I think `frames` is already a list of Pillow `Image` objects.

Reply:
It's needed because the `frames` array has `gif` images, they need to be saved into a new array of `webp` images. There's probably a more elegant solution when generating the `frames` array, though.

Reply:
`render_animation` could be improved with a gif/webp parameter, should increase performance considerably